orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -445,7 +445,7 @@ void RegisterTrainingOpSchemas() {
           "Constrain input shape to integer tensors.")
       .TypeConstraint(
           "T",
-          OpSchema::all_tensor_types(),
```

Comment:
>all_tensor_types [](start = 20, length = 16)

all_tensor_types doesn't include bfloat16? 

Looks like this should be updated in ONNX?

Reply:
yes i add bfloat16 to another list "all_tensor_types_bfloat16" in onnx.... to avoid potential issues.